kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/storage/PartitionLog.java

```diff
@@ -192,6 +195,10 @@ public boolean isInitialisationFailed() {
         return initFuture.isDone() && initFuture.isCompletedExceptionally();
     }
 
+    public void markAsUnloaded() {
```

Comment:
Is there a unit test available for this?

Reply:
Please check [here](https://github.com/streamnative/kop/pull/1966/files#diff-f7a7a4e368793ba72976ed229fe196eae7d34eede34240e28a8e18a0ac3e04f1R1450)